docker_test.go

```diff
@@ -1749,6 +1749,36 @@ func TestDockerContainerCopyFileToContainer(t *testing.T) {
 	}
 }
 
+func TestDockerCreateContainerWithFiles(t *testing.T) {
+	ctx := context.Background()
+	copiedFileName := "hello_copy.sh"
+
+	nginxC, err := GenericContainer(ctx, GenericContainerRequest{
+		ContainerRequest: ContainerRequest{
+			Image:        "nginx:1.17.6",
+			ExposedPorts: []string{"80/tcp"},
+			WaitingFor:   wait.ForListeningPort("80/tcp"),
+			Files: []ContainerFile{
+				{
+					HostFilePath:      "./testresources/hello.sh",
+					ContainerFilePath: "/" + copiedFileName,
+					FileMode:          700,
+				},
+			},
+		},
+		Started: true,
```

Comment:
Should we test this before the container is started?

```suggestion
		Started: false,
```

Reply:
@mdelapenya nice! thank you :) fixed
now I check files without a container running